developer_utilities/update_lists.py

```diff
@@ -34,6 +34,7 @@
 # pathlib.Path.match() paths to exclude from binary pruning
 PRUNING_EXCLUDE_PATTERNS = [
     'chrome/common/win/eventlog_messages.mc', # TODO: False positive textfile
+    'chrome/browser/resources/media/mei_preload/preloaded_data.pb',
```

Comment:
Do you know what this file is used for?

Reply:
Not really. MEI = media engagement index, which is about collecting analytics in order to determine whether to allow videos to auto-play

It does break the build if not included. It's only 14 bytes and doesn't contain anything interesting if printed with `protoc --decode_raw`. 

```
media/base/media_switches.cc
// Allows Media Engagement to use preloaded data to decide whether an origin has
// a high media engagement.

// - PreloadMediaEngagementData: enables a list of origins to be considered as
//   having a high MEI until there is enough local data to determine the user's
//   preferred behaviour.

```

Maybe in an official branded build they preload a larger file with real data in it?

Reply:
The proto is at `chrome/browser/media/media_engagement_preload.proto`

```
message PreloadedData {
  // The data in DAFSA format.
  optional bytes dafsa = 1;
}
```

Some more references:
`chrome/browser/media/media_engagement_preloaded_list.cc`
`chrome/browser/media/media_engagement_session.cc`
 `net/tools/dafsa`

Reply:
Alternatively the mei_preload files can be excluded with this patch:
```
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -304,7 +304,6 @@ if (!is_android && !is_mac) {
       }
 
       data_deps += [
-        "//chrome/browser/resources/media/mei_preload:component",
         "//third_party/widevine/cdm:widevinecdmadapter",
       ]
 
@@ -1239,7 +1238,6 @@ if (is_win) {
       ":widevine_cdm_library",
       "//build/config:exe_and_shlib_deps",
       "//chrome/app/nibs:chrome_xibs",
-      "//chrome/browser/resources/media/mei_preload:component_bundle",
     ]
 
     if (is_chrome_branded) {
```
I didn't notice anything unusual with such a build yet.

Reply:
I prefer excluding these files too

Reply:
Yeah, I don't see much benefit of including them in this case.

@squalus Could you include @xsmile's change as a new patch in the `common` base bundle (and also refresh any subsequent patches that also touch this file)?

Reply:
Nevermind I can merge now and do it.